datamodel/include/data_types/system/WindowConfig.hpp

```diff
@@ -33,6 +33,8 @@ struct WindowConfig {
     core::Dimensions size;
     visual::RGBAColor background_color;
     bool fullsreen = false;
+    bool process_input = false;
+    bool editor = true;
```

Comment:
If default value of `process_input` is `false`, then `editor` should be `false` too.